alchemiscale/tests/integration/interface/client/test_client.py

```diff
@@ -254,6 +639,18 @@ def test_cancel_tasks(
 
         assert canceled_sks_2 == [None]
 
+    def test_get_tasks_status(self):
+        ...
```

Comment:
This and below to be finished this PR or another?

Reply:
Another PR. I noticed that they were missing, but since we're going to be reworking the `status` methods and haven't implemented the `priority` methods yet, I added the placeholders.